### PR TITLE
Update site texts

### LINKED ISSUE #6: Review text content
The texts present in the text areas are temporary, it is necessary to add "permanent content" in these areas.

### PR DESCRIPTION
**Pull request by rluders**
I just updated the website texts.

Closes #6

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -62,21 +63,21 @@ const features = [
     topTitle: "Student Space",
     title: "Your space to learn",
     description:
-      "Lorem ipsum dolor sit amet consectetur adipisicing elit. Saepe eos, itaque perferendis eaque iure sequi cumque ea nostrum corrupti maiores fugit culpa praesentium velit quis deserunt distinctio natus atque dolore?",
+      "Customize it, get access to the courses based on your interest, and plan your learning path. The student space is the perfect place to improve the way that you learn while you follow your progress and get in touch with the instructor and other students.",
```
Comment (arianerocha) [inline src/pages/index.js:66]: can you mention about the platform adapting to the student?

**Review by arianerocha — CHANGES_REQUESTED** [review]

File: src/pages/index.js [hunk 2]
```diff
@@ -62,21 +63,21 @@ const features = [
...
     image: <img src="img/img01.svg" alt="Img 01" />,
   },
   {
     topTitle: "Instructor Space",
     title: "Your space to learn",
     description:
-      "Lorem ipsum dolor sit amet consectetur adipisicing elit. Saepe eos, itaque perferendis eaque iure sequi cumque ea nostrum corrupti maiores fugit culpa praesentium velit quis deserunt distinctio natus atque dolore?",
+      "The control of the content in your hands. Get statistics about the student progress, and course statistics. Find out how to improve the way that your content is delivered to the students while you plan your course or next class.",
```
Comment (arianerocha) [inline src/pages/index.js:73]: `The control of the content in your hands` or `The control of the content is in your hands`?

**Comment by rluders** [comment]
@arianerocha @mouracamila is it OK for you both? Can we merge it?

**Comment by rluders** [comment]
I merged the PR, if we need any other fix we can create a new issue, 'cause we didn't have any real text in the website, so... better to have some, even that not perfect, that doesn't had at all.

**Pull request merged**